src/components/PhotoPage/PhotoPage.js

```diff
@@ -108,21 +100,44 @@ const officers = [
 	},
 	{
 		role: 'Officer',
-		name: 'Furn Techalertumpai',
-		id: 'furn',
-		description: `Hi! I'm a fourth year CS major from Bangkok, Thailand 
-		(you can tell from my very long last name). I am a BIG fan of food and 
-		I like to listen to new jazz/R&B music. I will be co-directing Hack 
-		Kitchen with Raji this quarter so come say hi!!`
-	},
-	{
-		role: 'Officer',
-		name: 'Galen Wong',
+		name: 'Galen-senpai',
 		id: 'galen',
 		description: `I teach Web Dev a lot, feel free to ask me questions on 
 		JavaScript and React. I randomly organized a hot pot party at Kevin's 
 		place and they seem pretty satisfied.`
-	}
+	},
+	{
+		role: 'Officer',
+		name: 'Miles Wu',
+		id: 'miles',
+		description: `I’m a CS major passionate about hackathons and Powell library.
+		 Most of my experience deals with web/mobile dev. Catch me out here eating way too much food, drinking way too 
+		 much boba (@Jamie for enabling me), and vibing to chill music.`
+	},
+	{
+		role: 'Officer',
+		name: 'Alex Xia',
+		id: 'alex',
+		description: `Hi, I'm a first year CS major from Hong Kong. Aside from coding, 
```

Comment:
Some people use straight quotes `'` but others use curly quotes `’`. Make things `’` consistently.

Reply:
the curly quote was a problem that occurred when copy and pasting the descriptions. I have made things consistent and will stick with `'` and `"`